README.md

```diff
@@ -12,22 +12,43 @@ Installing your package manager of choice:
 
     pnpm i svelte-color-select
 
-Import into your component and bind `r`, `g`, and `b` values to it:
+Import into your component and pass an `{ r, g, b }` object to the `rgb` prop (with channels normalized 0–1):
 
 ```svelte
 <script lang="ts">
-	import ColorSelect from 'svelte-color-select`
+  import ColorSelect from 'svelte-color-select'
```

Comment:
I did switch to 2 spaces because GitHub doesn’t format tabs well. As a Svelte user I also use tabs in actual code but IMHO 2 spaces looks better in READMEs

Reply:
Yeah, definitely - there wouldn't be such a "tabs vs spaces" battle if the default size for tabs wasn't so bad!